addons/web/static/src/legacy/scss/bootstrap_review.scss

```diff
@@ -208,19 +208,37 @@ $o-navbar-nav-pills-link-border-radius: $nav-pills-border-radius !default;
     // body is white, the text will not be visible on modals with a white
     // background.
 
-    // Check if the body text color is light.
-    $-is-body-text-light: color-yiq($body-color) == $yiq-text-dark;
```

Comment:
Mention in the commit message that this work is a follow-up of https://github.com/odoo/odoo/commit/308b91c58b00300fd8dc52b9b4e2f7d1ab31f7b7 and actually improves what it did (the text muted stuff).